internal/test/hello/index.js

```diff
@@ -0,0 +1 @@
+export default 'Hello';
```

Comment:
why are there changes in both `internal/e2e` and `internal/test`? Seems like this file and the `internal/test/hello/package.json` don't do anything?

Reply:
whoops. didn't mean to commit those. initially had the hello module in test before moving it to e2e/rollup